### PR TITLE
Don't allow address zero recipient on signature minting

### PR DESCRIPTION
**Pull request by adam-maj**

### CODE REVIEW

File: packages/contracts-js/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "license": "Apache-2.0",
   "repository": "https://github.com/thirdweb-dev/js/tree/main/packages/contracts-js",
   "dependencies": {
-    "@thirdweb-dev/contracts": "3.1.8-0"
```
Comment (jnsdls) [inline packages/contracts-js/package.json:9]: did u mean to update this in this branch?

**Comment by joaquim-verges** [comment]
@adam-maj couple of build issues to fix

File: packages/contracts-js/package.json (hunk 1 above)
Comment (adam-maj) [reply packages/contracts-js/package.json:9]: yep, its intentional - this was to get the new contracts that don't allow zero address

**Comment by joaquim-verges** [comment]
merging this into my claim conditions branch

**Pull request closed without merging**